client/src/components/home/Home.js

```diff
@@ -1,13 +1,15 @@
 import { useSelector } from "react-redux";
 import { selectUser } from "../../features/auth/auth.selectors";
+import Statistics from "../statistics/Statistics";
 
 const Home = () => {
     const user = useSelector(selectUser);
-
+   
     return (
         <div className="text-center">
-            Hello { user.displayName }, this is the Home page placeholder
+            <Statistics/>
```

Comment:
You can move it to `Routing.js` with route `/statistics`, similar to how `/series/:id` is defined there
Then you can visit `/statistics` after login